src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,62 @@
+package lotto.domain;
+
+import static lotto.constants.ErrorMessage.DUPLICATED_LOTTO_NUMBER;
+import static lotto.constants.ErrorMessage.INVALID_LOTTO_SIZE;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import lotto.utility.validation.LottoNumberChecker;
+import lotto.vo.response.LottoResponse;
+
+public class Lotto {
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validateSize(numbers);
+        validateNumber(numbers);
+        validateDuplicate(numbers);
+        this.numbers = numbers;
+    }
+
+    private void validateSize(List<Integer> numbers) {
+        if (numbers.size() != 6) {
+            throw new IllegalArgumentException(INVALID_LOTTO_SIZE);
+        }
+    }
+
+    private void validateNumber(List<Integer> numbers) {
+        numbers.stream()
+            .forEach(number -> LottoNumberChecker.validate(number));
```

Comment:
(추천드려요!)
```suggestion
           numbers.forEach(LottoNumberChecker::validate);
```
컬렉션은 기본적으로 forEach를 지원해서 다음과 같이 변경할 수 있어요! 추가로 IDE의 도움을 받아서 메서드 참조도 활용해보면 좋을 것 같습니다 :)